cli/roger_build.py

```diff
@@ -82,6 +82,12 @@ def main(self, settingObj, appObj, hooksObj, dockerUtilsObj, dockerObj, args):
             else:
                 repo = data.get('repo', args.app_name)
 
+            build_args = {}
+            if 'build-args' in data:
+                if 'environment' in data['build-args']:
```

Comment:
Will build args be supported on something other than an environment by environment basis?
If environment builds are different, can we flag images to prevent `roger promote`?